pkgs/applications/misc/mapproxy/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "MapProxy";
+  version = "1.12.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "622e3a7796ef861ba21e42231b49c18d00d75f03eaf3f01a2b7687be7568e2ec";
+  };
+
+  prePatch = ''
+    substituteInPlace mapproxy/util/ext/serving.py --replace "args = [sys.executable] + sys.argv" "args = sys.argv"
```

Comment:
throwing away the location of the interpreter seems incorrect to me

Reply:
@FRidh , Please suggest what should be the alternate to this. This done to  get rid of the problem https://github.com/mapproxy/mapproxy/issues/424

Thanks in advance.